app/src/main/java/com/dinhtai/musicapp/model/data/HandleMusic.kt

```diff
@@ -0,0 +1,16 @@
+package com.dinhtai.musicapp.model.data
+
+import android.media.MediaPlayer
+import com.dinhtai.musicapp.model.entity.Music
+
+interface HandleMusic {
+    fun createMediaPlayer(path : String)
```

Comment:
```suggestion
    fun createMediaPlayer(path: String)
```
Dùng format code của android studio bằng `Ctrl + Alt + L` nhé